src/app/file-manager/filters/filesize-filter.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { TestBed, inject } from '@angular/core/testing';
+
+import { FilterSizePipe } from './filesize-filter';
+
+describe('SecurityService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [FilterSizePipe]
+    });
+  });
+  const BYTES = ' Bytes';
+  const KB = ' KB';
+  const MB = ' MB';
+  const GB = ' GB';
+  const TB = ' TB';
```

Comment:
It would be better to define these constants in `FilterSizePipe` so you can reference them in the test code and wouldn't need to duplicate them here.